tests/run/ctuple_6441.pyx

```diff
@@ -0,0 +1,133 @@
+# mode: run
+
+cimport cython
+
+cdef int b = 0
+
+cdef int has_side_effects() noexcept:
+    global b
+
+    b += 1
+    return b
+
+@cython.test_assert_path_exists("//IntNode")
+cdef int test_len_ctuple_var((int, int, int) a):
+    return len(a)
+
+@cython.test_assert_path_exists("//IntNode")
+cdef int test_len_ctuple_const():
+    return len(<(int, int, int)>(<int>1, <int>2, <int>3))
```

Comment:
Don't we  have an `IntNode` whatever happens, since there's either one for optimized `len()`, or three for `<int>1`, `<int>2`, `<int>3`?